### PR TITLE
Completed ES6 Destructuring exercise

### PR DESCRIPTION
**Pull request by maryalenn**
image

### CODE REVIEW

File: es6-destructuring/index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    ];
+
+    const book3 = library[0];
+    const book4 = library[1];
+    const book5 = library[2];
+    console.log('book3: ', book3);
+    console.log('book4: ', book4);
+    console.log('book5: ', book5);
+    const book6 = library[3];
+    console.log('book6: ', book6);
```
Comment (TimelikeClosure) [inline es6-destructuring/index.html]: Please use destructuring to get your values. Re-read the instructions and external documentation if you don't know what that is.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: es6-destructuring/index.html (hunk 1 above)
Comment (maryalenn) [reply es6-destructuring/index.html]: image

File: es6-destructuring/index.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    const book2 = {
+      title: 'Les Fleurs du mal',
+      author: 'Charles Baudelaire',
+      libraryID: 2345
+};
+    title = book2.title;
+    author = book2.author;
+    libraryID = book2.libraryID;
+
+    console.log(`The title of the book is ${title}, the author is ${author}, and the library id is ${libraryID}`);
```
Comment (TimelikeClosure) [inline es6-destructuring/index.html]: Please destructure `book1` and `book2`:

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: es6-destructuring/index.html (hunk 2 above)
Comment (maryalenn) [reply es6-destructuring/index.html]: image
image

File: es6-destructuring/index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+};
+const { title, author, libraryID } = book1;
+
+console.log(`The title of the book is ${title}, the author is ${author}, and the library id is ${libraryID}`);
+
+   const book2 = {
+     title: title2 = 'Les Fleurs du mal',
+     author: author2 = 'Charles Baudelaire',
+     libraryID: libraryID2 = 2345
+};
```
Comment (TimelikeClosure) [inline es6-destructuring/index.html]: `book2` should be defined in one statement and destructured in a different statement:

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: es6-destructuring/index.html (hunk 3 above)
Comment (maryalenn) [reply es6-destructuring/index.html]: Are you looking for something like this?

image

Comment (TimelikeClosure) [reply es6-destructuring/index.html]: @maryalenn It should be destructuring, but `book2` should be an object (as in the screenshot of the instructions), not an array.

**Comment by maryalenn** [comment]
image

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**